### PR TITLE
create animals table

### PR DESCRIPTION
**Pull request by mohamedelshamy55**
In this project, you will use a relational database to create the initial data structure for a vet clinic. You will create a table to store animals' information, insert some data into it, and query it.
Create tables in SQL.
Insert and query data in SQL.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_per_kg)
+VALUES ('Agumon',  '02/02/2020', '0', 'true', '10.23');
```
Comment (SajjadAhmad14) [inline data.sql]: The value for the date of birth of `Agumon` should be `2020/2/3`.

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_per_kg)
+VALUES ('Gabumon', '2018/11/15', '2', 'true', '8');
+
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_per_kg)
+VALUES ('Pikachu', '2021/01/17', '1', 'false', '15.04');
```
Comment (SajjadAhmad14) [inline data.sql]: The value for the date of birth of ``Pikachu`` should be  `2021/01/07`.

File: data.sql [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_per_kg)
+VALUES ('Devimon', '2017/04/12', '5', 'true', '11');
```
Comment (SajjadAhmad14) [inline data.sql]: The value for the date of birth of ``Devimon`` should be `2017/05/12`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Mohamed, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 3 above)
Comment (mohamedelshamy55) [reply data.sql]: thanks for your comments

**Review by AshakaE — APPROVED** [review]
### Hi @mohamedelshamy55   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mohamedelshamy55** [comment]
@SajjadAhmad14  and @AshakaE Thank you for finding time to review my project. I appreciate.

**Pull request merged**